### PR TITLE
Multi-battery support

### PR DESCRIPTION
**Pull request by sabrehagen**
Added support for multiple batteries on linux

### CODE REVIEW

File: bin/battery [hunk 1]
```diff
@@ -45,19 +43,23 @@ battery_status()
...
+            if [ -f "$BAT_PATH/energy_full" ]; then
+                naming="energy"
+            elif [ -f "$BAT_PATH/charge_full" ]; then
+                naming="charge"
+            elif [ -f "$BAT_PATH/capacity" ]; then
+                cat "$BAT_PATH/capacity"
+                return 0
+            fi
+            BAT_PERCENT=$(( 100 * $(cat $BAT_PATH/${naming}_now) / $(cat $BAT_PATH/${naming}_full) ))
+            BAT_TOTAL=$(( ${BAT_TOTAL-0} + $BAT_PERCENT ))
```
Comment (richo) [inline bin/battery]: If you hvae two batteries, will this read 200% when they're both full?

Comment (sabrehagen) [reply bin/battery]: Yes, then when printed the total is divided by the number of batteries to get the average: https://github.com/richo/battery/pull/3/files#diff-311d5b0abc17d5cff3ecafdb8c538b2cR26

Comment (richo) [reply bin/battery]: Ahh gotcha

**Comment by richo** [comment]
Thankyou!

**Pull request merged**